t/venom/pfservers/configurator/step1.yml

```diff
@@ -0,0 +1,112 @@
+name: step1 interfaces 
+version: 2
+testcases:
+- name: step1_dhcp_listener
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/configurator/config/interface/{{.configurator.dhcp_listener_interface}}'
+    ignore_verify_ssl: true
+    body: >-
+      {
+        "high_availability": 0,
+        "hwaddr": "{{.configurator.dhcp_listener_mac}}",
+        "id": "{{.configurator.dhcp_listener_interface}}",
+        "ifindex": "2",
+        "ipaddress": "{{.configurator.dchp_listener_ip}}",
+        "is_running": true,
+        "name": "{{.configurator.dhcp_listener_interface}}",
+        "netmask": "{{.configurator.dhcp_listener_mask}}",
+        "network_iseditable": false,
+        "type": "dhcp-listener"
+      }
```

Comment:
Could you try to limit payload to only include **mandatory** elements ?

Reply:
you mean " "network_iseditable": false, " ?